parts/2-ngi_pipeline/slides.md

```diff
@@ -1,10 +1,24 @@
-# NGI pipeline
+## TACA - Preprocessing
 
-## Intro to DNA sequencing
-We sequence the source code for life; DNA.
-How do you sequence DNA? What do you do with ACTG?
-Scale of data, automation, backup and clean up, why?
+First and foremost, data needs to go from the sequencer to a machine were the
+preprcessing is done. This is done in local servers because of two reasons.
```

Comment:
s/preprcessing/preprocessing/
Also as long as I'm nitpicking I'd write "on local servers." Maybe I should just PR my edits.